modules/wpa_screenshot_capture/src/Plugin/CaptureUtility/ScreenshotCaptureUtility.php

```diff
@@ -135,13 +213,26 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
       '#title' => $this->t('Browser background color'),
       '#description' => $this->t('Specify the browser background color in hexidecimal format. e.g. "#ffffff"'),
       '#default_value' => $this->configuration['background_color'],
+      '#states' => [
+        'visible' => [
+          'select[name="data[browser]"]' => ['value' => 'phantomjs'],
+        ],
+      ],
     ];
+    // As of now this only affects phantomjs. Support for this can be added to
+    // chrome but will require a PR/patch against the spatie/browsershot repo.
+    // This can come at a later time.
```

Comment:
Based on the comment here spatie/browsershot#130, if browser.js was slightly modified to allow for time-based waiting, we can maintain support in chrome.  

Depending on time, I may try to do open a PR doing this very thing, but may defer until later I get tied up.

Reply:
I ended up opening a PR and it got merged in: spatie/browsershot#139
This got included in v3.12.0 so I'm setting the library explicitly to that version.